### PR TITLE
Front page

### PR DESCRIPTION
**Pull request by Guy-deMessimy**

### CODE REVIEW

**Review by Marie-Garde — DISMISSED** [review]
Splendide !

File: package.json [hunk 1]
```diff
@@ -6,8 +6,10 @@
     "@testing-library/jest-dom": "^5.11.5",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "i": "^0.3.6",
```
Comment (LoicBrassart) [inline package.json]: `i` again ?

File: src/components/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      this.isMenuOpen = this.isMenuOpen.bind(this);
+     }; 
+
+     isMenuOpen () {
+      const isClose = !this.state.isOpen
+  this.setState({isOpen: isClose});
+      };
+
+render () {
+  console.log(this.state.isOpen)
```
Comment (LoicBrassart) [inline src/components/Sidebar.jsx]: Nope !

File: src/index.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
+
```
Comment (LoicBrassart) [inline src/index.js]: This new empty line is useless

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some Evils, please revise

**Review by Marie-Garde — DISMISSED** [review]
Nice !

**Comment by LoicBrassart** [comment]
I could review right now, but I'd have to do it again after conflicts are solved...
Could you please fix it and reassign me afterwards ?

**Review by Marie-Garde — DISMISSED** [review]
Ok

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -1,95 +1,301 @@
...
+const H4BIS = styled.h4`
+  font-size: 20px;
+  margin-bottom: 15px; 
+  @media ${device.xs} {
+   font-size: 10px;
+   margin-right: 5px;
+   margin-bottom: 3px; 
+  }`;
+
+const H4TER = styled.h4`
```
Comment (LoicBrassart) [inline src/components/Footer.jsx]: I still don't like this tag name, even though you explained it to me

File: src/components/Footer.jsx [hunk 5]
```diff
@@ -1,95 +1,301 @@
...
+`;
+const Socialicons = styled.ul`
+  margin-top: 15px;
+  @media ${device.xs} {
+   display: flex;
+   margin-top: 5px;
+  }
+`;
+
+const SocialLi = styled.li`
```
Comment (LoicBrassart) [inline src/components/Footer.jsx]: Ah ? A new Eveelution ? A very nice one ? =)

File: src/components/Sidebar.jsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+class Sidebar extends React.Component {
+  constructor(props)  {
+      super(props);
+      this.state = {
+          isOpen: false
+      }
+      this.isMenuOpen = this.isMenuOpen.bind(this);
+     }; 
+
+     isMenuOpen () {
```
Comment (LoicBrassart) [inline src/components/Sidebar.jsx]: This function name makes me think "Is my Menu open ?", not "I want to toggle my menu"
You should rename this "toggleMenu", it'd be clearer

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some minor evils

**Review by Marie-Garde — APPROVED** [review]
Nickel !

**Review by LoicBrassart — APPROVED** [review]
No more Evil ! :clap:

**Pull request merged**